msvc/libmonoruntime.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
What are the changes in this file about?

Reply:
VS adds UTF8 BOM mark to files when saved from within environment. I can manually remove that.